simulator/physics/Robot.cpp

```diff
@@ -245,10 +245,13 @@ void Robot::getWorldTransform(btTransform& chassisWorldTrans) const {
     _robotChassis->getMotionState()->getWorldTransform(chassisWorldTrans);
 }
 
-void Robot::radioTx(const Packet::RadioTx::Robot* data) {
-    velocity(data->body_x(), data->body_y(), data->body_w());
-    _controller->prepareKick(data->kick(), data->use_chipper());
-    _controller->prepareDribbler(data->dribbler());
+void Robot::radioTx(const Packet::Control* data) {
+    velocity(data->xvelocity(), data->yvelocity(), data->avelocity());
+    _controller->prepareKick(data->triggermode() == Packet::Control::IMMEDIATE
+                                 ? data->kcstrength()
+                                 : 0,
+                             data->shootmode());
```

Comment:
Should this be `shootmode() == CHIP`?  I'm not sure if the protobuf enum value is guaranteed to be the right thing here.